linux/LinuxProcess.c

```diff
@@ -280,7 +280,9 @@ static void LinuxProcess_writeField(const Process* this, RichString* str, Proces
       }
       xSnprintf(buffer, n, "%5lu ", lp->ctxt_diff);
       break;
-   case SECATTR: snprintf(buffer, n, "%-30.30s ", lp->secattr ? lp->secattr : "?"); break;
+   case SECATTR:
+      snprintf(buffer, n, "%-*.*s ", Process_fieldWidths[SECATTR], Process_fieldWidths[SECATTR], lp->secattr ? lp->secattr : "N/A");
+      break;
```

Comment:
If you keep CGROUP and CCGROUP in one line you might as well keep SECATTR in one line too …